common/app/common/commercial/CommercialProperties.scala

```diff
@@ -14,15 +14,12 @@ case class CommercialProperties(
   val isFoundationFunded: Boolean = branding(defaultEdition).exists(_.isFoundationFunded)
   def isSponsored(edition: Edition): Boolean = branding(edition).exists(_.isSponsored)
 
-  // Todo: remove this helper method when server-side ad targeting implemented before 2017-04-30
-  val surgeBuckets: String = adTargeting(defaultEdition).getOrElse("su", "")
-
   def branding(edition: Edition): Option[Branding] = for {
     editionBranding <- editionBrandings.find(_.edition == edition)
     branding <- editionBranding.branding
   } yield branding
 
-  def adTargeting(edition: Edition): Map[String, String] = {
+  def adTargeting(edition: Edition): Map[String, Either[String, Set[String]]] = {
```

Comment:
We want to distinguish between parameters that can only have a single value and those that can have multiple values, even if they actually have only one value.

The ADT is originally this: https://github.com/guardian/commercial-shared/blob/master/src/main/scala/com/gu/commercial/display/AdCallParamValue.scala#L8

I'll use this and then `stringify` it just before use in `JavascriptPage`.